ticketutil/ticket.py

```diff
@@ -67,19 +105,21 @@ def _create_requests_session(self):
             r.raise_for_status()
             logging.debug("Create requests session: Status Code: {0}".format(r.status_code))
             logging.info("Successfully authenticated to {0}".format(self.ticketing_tool))
+            return s
         # We log an error if authentication was not successful, because rest of the HTTP requests will not succeed.
         # Instead of using e.message, use e.args[0] instead to prevent DeprecationWarning for exception.message.
         except requests.RequestException as e:
-            logging.error("Error authenticating to {0}. No valid kerberos principal found.".format(self.auth_url))
+            logging.error("Error authenticating to {0}.".format(self.auth_url))
             logging.error(e.args[0])
-        return s
+            s.close()
 
```

Comment:
OK, I did asked @dmranck by mail... I see I should add you there too. I will ask Danny to respond here...